utils/appliance/__init__.py

```diff
@@ -1645,6 +1643,32 @@ def reset_automate_model(self):
             ssh_client.run_rake_command("evm:automate:reset")
 
 
+@navigate.register(IPAppliance)
+class LoggedIn(NavigateStep):
+    def step(self):
+        from cfme.login import login_admin
+        from utils.browser import browser
+        browser()
+        login_admin()
```

Comment:
How will we handle different users? I was wondering that this could take an optional param for a user and after that it would store it to some "world" - global context.